src/acquisition/flusurv/flusurv_update.py

```diff
@@ -68,122 +68,112 @@
   + initial version
 """
 
-# standard library
 import argparse
 
-# third party
-import mysql.connector
-
-# first party
 from delphi.epidata.acquisition.flusurv import flusurv
-import delphi.operations.secrets as secrets
 from delphi.utils.epidate import EpiDate
 from delphi.utils.epiweek import delta_epiweeks
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session
+
+from ...server._config import (
+    SQLALCHEMY_DATABASE_URI,
+    SQLALCHEMY_ENGINE_OPTIONS
+)
+from .models import FluSurv
 
+engine = create_engine(SQLALCHEMY_DATABASE_URI, **SQLALCHEMY_ENGINE_OPTIONS)
```

Comment:
chat GPT recommends to do something like this 
```python
from sqlalchemy import create_engine

def get_engine():
    engine = create_engine('postgresql://user:password@localhost/mydatabase')
    return engine

with get_engine().connect() as conn:
    result = conn.execute("SELECT * FROM mytable")
    for row in result:
        print(row)
```
instead of global variables

but my suggestion is to make even deeper
```python
db = SQLAlchemy(metadata=meta)
ma = Marshmallow()
bp = Blueprint('bp', __name__, url_prefix='/api')
migrate = Migrate(db)


def create_app(config_object=Config):
    ''' Flask Application Factory

    Docs:
        https://flask.palletsprojects.com/en/2.0.x/patterns/appfactories/
    '''

    flask_app = Flask(__name__)
    flask_app.config.from_object(config_object())

    # init db and serializer
    db.init_app(flask_app)
    ma.init_app(flask_app)
    migrate.init_app(flask_app, db)

    # register app blueprints
    import app.event_impact.blueprints  # noqa

    # register blueprints
    flask_app.register_blueprint(bp)

    return flask_app
```

but it will affect a lot

Reply:
checking my understanding -- you're recommending we use Flask on the acquisition side as well? We currently only use it for the API server.